### PR TITLE
Revert changes to use `<RestKit/...>` style headers

### LINKED ISSUE #2356: RestKit doesnt build in version 0.26
Maybe this is my problem, as I would assume its working for some people.

I have updated to 0.26 today, I am using `use_frameworks!` in my Podfile (cocoapods 0.39)

It fails in `CoreData.h` on `#import <RestKit/CoreData/RKEntityMapping.h>` with file not found

I have tried cleaning, removing `Pods/`, no luck getting this to build.

### PR DESCRIPTION
**Pull request by mcfedr**
Reverting these commits means that RestKit can work with Cocoapods 0.39.0 when using `use_frameworks` which with the rise of Swift is quite important.

As far as I can tell using `<RestKit/...>` headers inside of RestKit itself doesn't make much sense, as the RestKit framework hasn't been built yet.

There are still problems with Cocoapods 1-beta, I think these will need to be addressed in AFNetworking.

Fixes #2356

### CODE REVIEW

File: Code/CocoaLumberjack/RKLumberjackLogger.m [hunk 1]
```diff
@@ -24,7 +24,7 @@ + (DDLogLevel)ddLogLevelFromRKLogLevel:(_RKlcl_level_t)rkLevel
         case RKLogLevelDebug:    return DDLogLevelDebug;
         case RKLogLevelTrace:    return DDLogLevelVerbose;
     }
-
+
```
Comment (segiddins) [inline Code/CocoaLumberjack/RKLumberjackLogger.m]: what's with these whitespace changes?

**Comment by segiddins** [comment]
👍 other than the whitespace regressions

File: Code/CocoaLumberjack/RKLumberjackLogger.m (hunk 1 above)
Comment (mcfedr) [inline Code/CocoaLumberjack/RKLumberjackLogger.m]: I have just used `git revert ...` and these must have been cleaned up by one of the original commits
I'll make a new commit to clean up

**Comment by segiddins** [comment]
@mcfedr should I release a 0.26.1 with these changes?

**Comment by mcfedr** [comment]
That would be great

**Comment by erikackermann** [comment]
@segiddins is a 0.26.1 release still pending?

**Comment by segiddins** [comment]
@erikackermann eventually, haven't gotten around to it

**Pull request merged**